algorithms/kernel/brownboost/brownboost_train_impl.i

```diff
@@ -320,7 +327,11 @@ services::Status BrownBoostTrainKernelNew<method, algorithmFPType, cpu>::compute
     }
     }
 
-    if (alpha) { daal::services::daal_free(alpha); }
+    if (alpha) 
```

Comment:
FYI (no changes are required):
free() checks ptr for nullptr and does nothing if it is null ptr.